### PR TITLE
BLOCKCHAIN-364 - LLM monthly vault mint instead of yearly

### PR DESCRIPTION
**Pull request by kacperzuk-neti**

### CODE REVIEW

File: frame/llm/src/tests.rs [hunk 1]
```diff
@@ -52,7 +52,8 @@ fn releases_on_future_block() {
 		System::set_block_number(next_block);
 		LLM::on_initialize(next_block);
 
-		let expected_release = vault_before / 10;
+		let factor = <Test as crate::Config>::InflationEventReleaseFactor::get();
```
Comment (MichalLustykNeti) [inline frame/llm/src/tests.rs:55]: let's add config at the top of file

Comment (kacperzuk-neti) [reply frame/llm/src/tests.rs]: done.

**Comment by kacperzuk-neti** [comment]
Over "Year 3" it releases `5670000.027986572980` LLM instead of `5670000` LLM due to rounding issues. Is that fine?

**Comment by DorianSternVukotic** [comment]
> Over "Year 3" it releases `5670000.027986572980` LLM instead of `5670000` LLM due to rounding issues. Is that fine?

yes

**Review by MichalLustykNeti — APPROVED** [review]

**Comment by DorianSternVukotic** [comment]
So just to be super clear

How this works is that every time a release is triggered, it adds 1 month to the last release.

This means that if we forget to release (or do an upgrade first time) , we can trigger it many times until the last release counter reaches blocks that are produced in the last month, after which we get Gottawait or something similar?

**Comment by kacperzuk-neti** [comment]
> So just to be super clear
> 
> How this works is that every time a release is triggered, it adds 1 month to the last release.
> 
> This means that if we forget to release (or do an upgrade first time) , we can trigger it many times until the last release counter reaches blocks that are produced in the last month, after which we get Gottawait or something similar?

Regarding LastRelease - yes. But we don't trigger it manually - `maybe_release()` function is triggered automatically on every block. So when we upgrade, it will do one release on each block until the counter reaches the expected date.

**Pull request merged**